src/tigerbeetle/cli.zig

```diff
@@ -145,7 +145,8 @@ pub fn parse_args(allocator: std.mem.Allocator) !Command {
     var verbose: ?bool = null;
 
     var args = try std.process.argsWithAllocator(allocator);
-    defer args.deinit();
+    // NOTE: do not deinit() as we return the path and this could free/invalidate it.
+    // defer args.deinit();
```

Comment:
Now that ArgsIterator owns the data, it semes we can remove `args_allocated` and use

```
    pub const Start = struct {
        args_allocated: std.process.ArgIterator,
```

?